### ISSUE TITLE
LLVM with RVV 0.7.1 support

### ISSUE THREAD

**Issue by zaazad** [plctlab/llvm-project#32]
Hi, 

I need LLVM support for RVV 0.7.1 to cross compile for Nezha board. 
I could find support for older versions such as RVV0.8 and 0.9 in your repo. 
Is there any branch with support for RVV 0.7.1?

Thank you!

**Comment by ChunyuLiao** [comment]
Sorry, we don't  support RVV 0.7.1.

**Comment by joy2myself** [comment]
> There the branch 0.7.1 is not referring to rvv?

Yes, it is. But branch 0.7.1 is a development branch two years ago. We only added some rvv-0.7.1 instructions to the assembler. The llvm version corresponding to this branch is also very old and does not support auto-vectorization. Therefore, the branch probability cannot meet the existing use requirements.
